encfs/encfssh

```diff
@@ -6,16 +6,17 @@
 # Contributed by David Rosenstrauch.
```

Comment:
Strongly opposed. See [BashFAQ #105](http://mywiki.wooledge.org/BashFAQ/105) and [BashFAQ #107](http://mywiki.wooledge.org/BashFAQ/107).